src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,13 @@
-export const UserInfo = () => {};
+import { User } from '../../types/User';
+
+type Props = {
+  user: User | null;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  return user && (
```

Comment:
```suggestion
  return 
```
No need to implement an additional check if you have conditional rendering